.eslintrc.js

```diff
@@ -1070,6 +1070,12 @@ module.exports = {
         '@typescript-eslint/no-duplicate-imports': ['error'],
       },
     },
+    {
+      files: ['x-pack/plugins/cases/public/**/*.{js,mjs,ts,tsx}'],
```

Comment:
How do you feel about ignoring test files? I think we could add this: https://eslint.org/docs/user-guide/configuring/ignoring-code#ignorepatterns-in-config-files

Reply:
good idea.

Reply:
@jonathan-buttner I excluded test files and removed the changes from them.